pkgs/desktops/gnome/apps/gnome-clocks/default.nix

```diff
@@ -45,9 +45,6 @@ stdenv.mkDerivation rec {
     desktop-file-utils
     libxml2
     gobject-introspection # for finding vapi files
```

Comment:
presumably this gobject-introspection can be removed while you're here too, going by the comment.